module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,62 @@
+#include "AppendNewRecipe.hpp"
+
+#include <algorithm>
+#include <fstream>
+
+bool AppendNewRecipe(std::vector<std::string> steps,
+                     const std::list<std::string>& ingredients,
+                     const std::deque<std::pair<size_t, char>>& amount) {
+    std::ofstream file("recipes.txt", std::ios::app);
```

Comment:
I think file path should be const variable.

Reply:
+1